From 83ac3a86c77c586f4463b15e60a7c02e688dcf3b Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 18 Nov 2021 10:44:00 +0100
Subject: [PATCH] Revert "turris-timezone: temporary fix for l18n"

l18n developers did some Python magic, because with a recent update to
2021.3.1 version, they add Python 3.10 compatibility, which
unfortunately requires having an installed wheel before running setup.py. 
Also, turris-timezone requires l18n otherwise it is not compiled.

The issue was with OpenWrt 19.07, which uses always the latest version for 
host(side) packages, and luckily for us Python OpenWrt maintainers
(commodo, jefferyto) added the option to have pinned host dependencies for OpenWrt since 21.02.

This is done in the packages feed and lang/host-pip-requirements folder.

This reverts commit a6670e6c5366b4dd6f64a95f1ef195c773e28bd4.
---
 utils/turris-timezone/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/turris-timezone/Makefile b/utils/turris-timezone/Makefile
index b10ffecb8..99279170a 100644
--- a/utils/turris-timezone/Makefile
+++ b/utils/turris-timezone/Makefile
@@ -15,6 +15,7 @@ PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/foris-controller/$(PKG_NAME)/-/archive/v$(PKG_VERSION)/
 PKG_HASH:=0042a108d81b56c414e229638d432c15acc75ed1591a68ab213ad21db657441c
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-v$(PKG_VERSION)
+HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=l18n
 
 PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
 PKG_LICENSE:=GPL-3.0-or-later
-- 
2.30.2

